klippy/chelper/trapq.c

```diff
@@ -206,7 +204,7 @@ trapq_set_position(struct trapq *tq, double print_time
                    , double pos_x, double pos_y, double pos_z)
 {
     // Flush all moves from trapq
-    trapq_finalize_moves(tq, NEVER_TIME);
+    trapq_finalize_moves(tq, NEVER_TIME, 0);
```

Comment:
This would introduce a subtle change in behavior.  All places that fully cleared the history should continue to fully clear the history - thus `trapq_finalize_moves(tq, NEVER_TIME, NEVER_TIME);`